config/proto/targets.proto

```diff
@@ -4,29 +4,37 @@ package hermes;
 
 option go_package = "github.com/googleinterns/step224-2020/targets/proto";
 
+// Definition of a Target, a storage system to be probed by a specific probe.
 message TargetsDef {
   enum TargetSystem {
     GOOGLE_CLOUD_STORAGE = 0;
     CEPH = 1;
   }
 
   enum ConnectionType {
-   HTTP = 0;
-   HTTPS = 1;
+     HTTP = 0;
+     HTTPS = 1;
   }
 
   required TargetSystem target_system = 1;
-  ConnectionType connection_type = 2;
+  required ConnectionType connection_type = 2;
```

Comment:
Also, apparently, required fields are things to avoid when using proto2 [1]. ¯\_(ツ)_/¯

[1] https://developers.google.com/protocol-buffers/docs/style#things_to_avoid

Reply:
I think we might switch to Proto3 (dependent on confirmation with Alicja), which does not support required, so I'll review these when confirmed.

Reply:
We are switching to Proto3!